backend/src/apiserver/model/pipeline.go

```diff
@@ -29,7 +29,7 @@ const (
 type Pipeline struct {
 	UUID           string `gorm:"column:UUID; not null; primary_key"`
 	CreatedAtInSec int64  `gorm:"column:CreatedAtInSec; not null"`
-	Name           string `gorm:"column:Name; not null; unique"`
```

Comment:
We removed the `unique` here for name but we should change the tuple(Namespace, Name) unique.

Reply:
@capri-xiyue This has been taken care of in the client manager